web/channels/retro_channel.ex

```diff
@@ -76,6 +76,18 @@ defmodule RemoteRetro.RetroChannel do
     {:noreply, socket}
   end
 
+  def handle_in("submit_vote", %{"id" => id}, socket) do
+    idea = Repo.get(Idea, id)
```

Comment:
based on the handler i see in the retro, where we're only relying on `id` and `vote_count` attributes, there shouldn't be any need to preload the user here, which will save us a database hit. 

The select then update queries can be consolidated into an update query using the following syntax:

```elixir
    query = from i in Idea, where: i.id == ^id
    {_row_count, [updated_idea]} = Repo.update_all(query, [inc: [vote_count: 1]], returning: true)
```

Not the prettiest thing in the world, but gets the job done in one go.